src/fsharp/fsi/fsiAnyCPU.exe.config

```diff
@@ -10,7 +10,7 @@
           culture="neutral"/>
         <bindingRedirect
           oldVersion="2.0.0.0-4.4.1.0"
-          newVersion="4.4.1.9055"/>
+          newVersion="4.4.1.0"/>
```

Comment:
I'm not entirely clear on the ramifications of this change particularly for existing installations of Visual Studio.  Perhaps it's benign because nothing is GAC'd in the new world.  Just wondering...